packages/vapor/scss/tables/table.scss

```diff
@@ -239,3 +239,34 @@ $row-style: (
         fill: $dark-medium-grey;
     }
 }
+
+.blacklistCardContainer {
```

Comment:
I would probably scope the changes inside `.table` with a more generic classname (`mod-card-list` or something in the same vein)